README.md

```diff
@@ -325,13 +325,27 @@ end
 ## Releasing
 
 After following the steps from the [Developing section](#developing) to build
-and test the gem, and merging all changes to the `master` branch, first bump
-the version number by editing
-[`lib/jekyll_pages_api_search/version.rb`](lib/jekyll_pages_api_search/version.rb).
-Then run `bundle exec rake release` to build the gem, tag the head commit in
-`git`, push the branch and tag to GitHub, and ultimately push the release to 
-[`jekyll_pages_api_search` on
-RubyGems.org](https://rubygems.org/gems/jekyll_pages_api_search).
+and test the gem: 
+
+1. Ensure all changes for the release have already been merged all into the
+   `master` branch.
+   
+1. Bump the version number by editing
+   [`lib/jekyll_pages_api_search/version.rb`](lib/jekyll_pages_api_search/version.rb).
+
+1. Commit the version number update directly to the `master` branch, replacing
+   `X.X.X` with the new version number:
+   ```sh
+   $ git add lib/jekyll_pages_api_search/version.rb
+   $ git commit -m 'Bump to vX.X.X'
+   ```
```

Comment:
I suppose you could commit and comment in one line, yeah. I just tend to keep them separate. But the `bundle exec rake release` will handle the tag and push.

For node modules, yeah, you have to do `git tag X.X.X && git push --tags`, but that's for a different README.